### PR TITLE
{bio}[foss/2022a] ont-remora v2.0.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

* [x] #17179
* [x] #17181

### CODE REVIEW

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/o/ont-remora/ont-remora-2.0.0-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('Python', '3.10.4'),
+    ('Arrow', '8.0.0'),
+    ('h5py', '3.7.0'),
+    ('python-parasail', '1.3.3'),
+    ('Pysam', '0.20.0'),
+    ('PyTorch', '1.12.1'),
```
Comment (branfosj) [inline easybuild/easyconfigs/o/ont-remora/ont-remora-2.0.0-foss-2022a-CUDA-11.7.0.eb:22]:
```suggestion
    ('PyTorch', '1.12.1', versionsuffix),
```

Comment (branfosj) [reply easybuild/easyconfigs/o/ont-remora/ont-remora-2.0.0-foss-2022a-CUDA-11.7.0.eb]: irrelevant, as I've split the CUDA variant out of this PR

**Comment by verdurin** [comment]
@branfosj note the bleating about `PyTorch` deps.

**Comment by branfosj** [comment]
Newer versions/toolchains merged, so closing this.

**Pull request closed without merging**